backend/tests/test_logout.py

```diff
@@ -2,19 +2,32 @@
 import pytest
 
 
-def test_logout_user_with_valid_access_token(client, access_token):
+def test_logout_user_with_valid_access_token(client, tokens):
     """Test logout with fresh access token."""
     rv = client.delete('/auth/logout/',
                        headers={'Authorization': 'Bearer {}'
-                                .format(access_token)})
+                                .format(tokens["access"])})
     response = rv.get_json()
     assert rv.status_code == HTTPStatus.OK
     assert response['msg'] == 'Successfully logged out'
 
 
-def test_logout_user_without_token(client):
+@pytest.mark.parametrize('url', ['/auth/logout/', '/auth/revoke-refresh-token/'])
```

Comment:
That will change when #184 will get merged, but besides that, it's a pretty good change.